app/api/recipe.py

```
@@ -0,0 +1,51 @@
+from app.core.dependencies import CurrentUserId, DBSession
+from app.schemas.base import ApiResponse, ok
+from app.schemas.recipe import AnalyzeRequest, RecipeResponse, SaveRequest
+from app.services.recipe import RecipeService
+
+router = APIRouter(prefix="/recipe", tags=["recipe"])
+
+
+@router.post("/analyze")
+async def analyze_recipe(
+    body: AnalyzeRequest,
+    user_id: CurrentUserId,
```

Ruff (0.14.14)

[warning] Unused function argument: `user_id`

(ARG001)